MapboxCoreNavigation/RouteController.swift

```diff
@@ -689,33 +705,12 @@ extension RouteController: CLLocationManagerDelegate {
         let radius = max(reroutingTolerance, location.horizontalAccuracy + RouteControllerUserLocationSnappingDistance)
         let isCloseToCurrentStep = newLocation.isWithin(radius, of: routeProgress.currentLegProgress.currentStep)
         
-        guard !isCloseToCurrentStep else { return true }
-        
-        // Check to see if the user is moving away from the maneuver.
```

Comment:
This is now effectively replaced with the `isCourseIsOnRoute` check below.

Reply:
Maybe we should rename this `isCourseIsOnRoute` -> `isCourseOnRoute` 😸